### ISSUE TITLE
Feature Request: Set Sample in Scene as ONE SHOT

### ISSUE THREAD

**Issue by artqcid** [raspy135/Qun-mk2#9]
I would really love a Parameter where you can decide in a Scene if a Sample of a Track should looped or only played once. This would give much more possibilites to Program Fills or Breaks. What do you think?

**Comment by raspy135** [comment]
It's basically looper that will synched with sequencer. Maybe you can use empty scene to stop looper if you don't like manual looper stop.

**Comment by artqcid** [comment]
Yes it's possible to stop manually or change the scene. But then you have to keep your Hand on the Unit and wait for the Moment on when you want to stop it. That said, it needs additional action on time. If it would stop from alone then you do t need your hand on the Unit on the end of the sequence. Please consider that the Qun is used in combination with other gear. With One Shot you can start the Break, but have your Hands free to do Action at the End of the Break.

**Comment by artqcid** [comment]
So as far I understand you there is technical only a way to stop or play all three Tracks? Then I understand that there is no chance to introduce one shot. But thanks for your time and answer.
